### PR TITLE
Add protobuf to requirements.

### PR DESCRIPTION
**Pull request by nwbirnie**
Idea is to avoid issues like #412 in future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ This project hosts the Java client library for the Google Ads API.
 
   * Java 1.8+
   * Maven 3.0+
+  * Protobuf 3.12.2+
```
Comment (jradcliff) [inline README.md:15]: Are you also seeing gRPC dependencies as an issue, or is protobuf by far the most common problem?

Comment (jradcliff) [inline README.md:15]: Suggest making "Protobuf" a link to the protobuf library

Comment (jradcliff) [inline README.md:15]: How frequently do you anticipate this version will need to change? Wondering if it makes sense to automate keeping this in sync with poms, like you do with the library version on line 22.

Comment (nwbirnie) [reply README.md:15]: I'm not aware of a specific lower bound for gRPC. guava is a more common source of issues. I'll add this to the line up as well.

Comment (nwbirnie) [reply README.md]: Done

Comment (nwbirnie) [reply README.md]: Good point, I'll add this to the release tasks for the next release.

**Review by devchas — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jradcliff) [reply README.md]: I'm not trying to complicate things (honest!), but would it be better if this was a separate section, or a "Dependencies" subsection of "Requirements"?

The protobuf and guava dependency requirements are different from the Java runtime requirement. The former only come into play if the user _already_ includes protobuf or guava as a transitive or direct dependency in their project. If not, google-ads-java will work just fine. In contrast, if the user tries to use Java 1.5 :tiger: , it's likely they'll run into issues.

Perhaps something like:

```
## Compatible dependencies

*  protobuf 3.12.2+
*  guava x.y.z+
```

Comment (nwbirnie) [reply README.md]: Sounds good. Updated.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ This project hosts the Java client library for the Google Ads API.
   * Java 1.8+
   * Maven 3.0+
 
+## Compatible dependencies
+  * [Protobuf](https://github.com/protocolbuffers/protobuf) 3.12.2+
```
Comment (jradcliff) [inline README.md:17]: nit: add a blank line after the header to match style in rest of the doc

**Review by jradcliff — APPROVED** [review]
One tiny nit, else LGTM

**Pull request merged**